contrib/question_converters/canvas/quiz2pl.py

```diff
@@ -102,6 +103,67 @@ def clean_question_text(text: str) -> str:
     return text
 
 
+def image_file_extension(content_type: str) -> str:
+    match content_type:
+        case "image/x-icon":
+            return "ico"
+        case "image/svg+xml":
+            return "svg"
+        case _:
+            return content_type.split("/")[1]
+
+
+def handle_images(question_dir: str, text: str) -> str:
+    # Links to images will still point to Canvas. We need to download them and
+    # replace them with `pl-figure` elements.
+    #
+    # We use regex instead of a proper HTML parser because we want to limit this
+    # script to only using the Python standard library.
+    image_idx = 1
```

Comment:
```suggestion
    image_counter = count(1)
```
Nit: then the for loop can read `for image_idx, match in zip(image_counter, ...):`

Reply:
This actually has different behavior: if we hit the `continue` case, we'll end up skipping indexes in the file names.

Reply:
Ah neat! Done in dc7b038